Rise Media Player Dev/Windows/NowPlayingPage.xaml.cs

```diff
@@ -151,7 +151,7 @@ private async void PlaybackSession_PositionChanged(MediaPlaybackSession sender,
 
         private IAsyncAction UpdateCurrentLyricAsync(TimeSpan playerPosition)
         {
-            var lyricsItem = _lyrics?.FirstOrDefault(item => item.TimeSpan.TotalSeconds - playerPosition.TotalSeconds >= 0);
+            var lyricsItem = _lyrics?.LastOrDefault(item => item.TimeSpan.TotalMilliseconds < playerPosition.TotalMilliseconds);
```

Comment:
```suggestion
            var lyricsItem = _lyrics?.LastOrDefault(item => item.TimeSpan.TotalSeconds < playerPosition.TotalSeconds);
```
Change it to seconds maybe?